packages/mgt-chat/src/statefulClient/GraphNotificationUserClient.ts

```diff
@@ -259,22 +259,29 @@ export class GraphNotificationUserClient {
 
       if (!subscription) {
         try {
-          subscription = await this.createSubscription(this.currentUserId);
+          subscription = await this.createSubscription(currentUserId);
         } catch (e) {
           subscription = undefined;
           isRenewalInError = true;
           error(e);
 
-          // rather than 3 seconds, back off to 10 seconds if we get a 403
-          if ((e as { statusCode?: number }).statusCode === 403) {
-            nextRenewalTimeInMs = 10 * 1000;
+          const err = e as { statusCode?: number; message: string };
+          if (err.statusCode === 403) {
+            // rather than 3 seconds, back off to 10 seconds if we get a "403" which should really be a 429 - this happens when we reached subscription limit of 10
+            if (err.message.indexOf('has reached its limit') > 0) {
```

Comment:
should be `!== -1` if checking for presence.
ES6 added .includes(), that may work as well

Reply:
the effects are the same, can you describe what cases the above doesn't cover?

Reply:
if that substring somehow starts at index 0 its not covered

Reply:
it is not expected for that 'has reached its limit' to actually start at 0 was my point to let it be > 0 vs -1

Reply:
yeah my point is that the intent of the line is to see if that substring is present. It's a small issue, because that message probably won't change, but it does cover an edge case and make your intent crystal clear

Reply:
I wanted to be clear that if the substring appears at the start, then I suspect something is wrong, because I expect the string to be not at the start, so I wanted to be clear on that.